### PR TITLE
Lesson 35 1 2 

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**
1) added useEffevt to fetch homes
2) added useEffect to fetch and get available homes from composed url - does not work. Composed url can not receive value (filter does not work).

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,26 +1,98 @@
...
+  const [AvailableIsOpen, setAvailableIsOpen] = useState(false);
+
+  const handleValue = (e) => {
+    setValue(e.target.value);
+  };
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setAvailableIsOpen(true);
+    // setDestination(searchMatches(homes, value));
```
Comment (Marinarium) [inline src/components/App/App.js]: delete comments with unused code please

File: src/components/GetHomes/GetHomes.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, {useState, useEffect} from 'react';
+import Card from "../Card/Card";
+import './GetHomes.css';
+
+const GetHomes = () => {
+  const homesUrl = 'https://fe-student-api.herokuapp.com/api/hotels/popular';
+  const[homes, setHomes]= useState([]);
+
+
```
Comment (Marinarium) [inline src/components/GetHomes/GetHomes.jsx]: excess space

File: src/components/GetHomes/GetHomes.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, {useState, useEffect} from 'react';
+import Card from "../Card/Card";
+import './GetHomes.css';
+
+const GetHomes = () => {
+  const homesUrl = 'https://fe-student-api.herokuapp.com/api/hotels/popular';
```
Comment (Marinarium) [inline src/components/GetHomes/GetHomes.jsx]: add to constants please

File: src/components/App/App.js [hunk 4]
```diff
@@ -1,26 +1,98 @@
...
+              <Button
+                className="button-lg"
+                title="Search"
+                onClick={handleClick}
+              />
+            </div>
+          </form>
+        </div>
+      </MainPage>
+      {AvailableIsOpen && (
```
Comment (Marinarium) [inline src/components/App/App.js:80]: shouldn't containers be in MainPage?

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js]: Done.

File: src/components/App/App.js (hunk 4 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js:80]: MainPage is a page with a  bright colored background. It is a separate element. As soon as it was like this  in the previous project, I decided to leave it as it was.

File: src/components/GetHomes/GetHomes.jsx (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/components/GetHomes/GetHomes.jsx]: Done

File: src/components/GetHomes/GetHomes.jsx (hunk 3 above)
Comment (LudmilaZhuleha) [reply src/components/GetHomes/GetHomes.jsx]: Done

**Pull request merged**